app/src/main/java/org/mozilla/fenix/components/BackgroundServices.kt

```diff
@@ -65,21 +66,15 @@ class BackgroundServices(
         Build.MODEL
     )
 
-//  private val serverConfig = ServerConfig.release(CLIENT_ID, REDIRECT_URL)
-//  TODO replace with release channel ^^
-    private val serverConfig = ServerConfig("https://oauthchannel2.dev.lcip.org", CLIENT_ID, REDIRECT_URL)
+    private val serverConfig = ServerConfig(FeatureFlags.webChannelConfiguration.contentUrl, CLIENT_ID, REDIRECT_URL)
     private val deviceConfig = DeviceConfig(
         name = defaultDeviceName(context),
         type = DeviceType.MOBILE,
 
         // NB: flipping this flag back and worth is currently not well supported and may need hand-holding.
         // Consult with the android-components peers before changing.
         // See https://github.com/mozilla/application-services/issues/1308
-        capabilities = if (FeatureFlags.sendTabEnabled) {
-            setOf(DeviceCapability.SEND_TAB)
-        } else {
-            emptySet()
-        }
+        capabilities = setOf(DeviceCapability.SEND_TAB)
```

Comment:
Let's leave the `sendTabEnabled` flag in place.

Reply:
In Fenix, master they don't have it

Reply:
oh, ok! cool.